tests/integration/cqlengine/model/test_model.py

```diff
@@ -105,6 +107,25 @@ class table(Model):
 
         drop_keyspace('keyspace')
 
+    def test_column_family(self):
+        class TestModel(Model):
+            k = columns.Integer(primary_key=True)
+
+        from cassandra.cqlengine import models
```

Comment:
Good question. I think I put it here after looking that ``DEFAULT_KEYSPACE`` access in ``cqlengine.connection``. There the module import is imported inside the functions to avoid circular references with ``cqlengine.model``. 
Here in the test that is not an issue. I'll move it out.